### ISSUE TITLE
small bug in sc.pp.filter_cells()

### ISSUE THREAD

**Issue by LiBuchauer** [scverse/scanpy#609]
Hi,
trying to use `inplace=False` in `pp.filter_cells()` I found a small bug:
```python
if isinstance(data, AnnData):
    adata = data.copy() if copy else data
    cell_subset, number = materialize_as_ndarray(filter_cells(adata.X, min_counts, min_genes, max_counts, max_genes))
    if not inplace:
        return gene_subset, number
```
Should probably be `return cell_subset`.

**Comment by ivirshup** [comment]
That definitely looks like a bug. Would you want to open a PR for that?

**Comment by LiBuchauer** [comment]
I opened one, #611, its my first time (scary), hope I didn't do anything majorly stupid ;)

**Comment by ivirshup** [comment]
Looks great, thanks!

Closed via #611.

**Comment by flying-sheep** [comment]
> hope I didn't do anything majorly stupid

You did well, good job! You can check the [“Files changed”](https://github.com/theislab/scanpy/pull/611/files) tab of your PR if everything looks like you intended :smile:
